src/HSE/All.hs

```diff
@@ -113,13 +112,9 @@ ghcExtensionsFromParseFlags ParseFlags{enabledExtensions=es, disabledExtensions=
 ghcFixitiesFromParseFlags :: ParseFlags -> [(String, Fixity)]
 ghcFixitiesFromParseFlags = map toFixity . fixities
 
+-- GHC fixities declared at the top level of a module.
 ghcFixitiesFromModule :: Located (HsModule GhcPs) -> [(String, Fixity)]
-ghcFixitiesFromModule (L _ (HsModule _ _ _ decls _ _)) = concatMap f decls
-  where
-    f :: LHsDecl GhcPs -> [(String, Fixity)]
-    f (L _ (SigD _ (FixSig _ (FixitySig _ ops (GHC.Fixity _ p dir))))) =
-          fixity dir p (map rdrNameStr' ops)
-    f _ = []
+ghcFixitiesFromModule = fixitiesFromModule
```

Comment:
Can we just inline this everywhere, so there is no more ghcFixitiesFromModule.

Reply:
I can do that. Stand by.